### ISSUE TITLE
I got startup error

### ISSUE THREAD

**Issue by budiz1** [FreeCAD/FreeCAD#15964]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

When i start the latest Freecad weekly build, i get an error that says (in the picture below).

I dont know if that is my computers problem or freecads as this error never showed up before.

<img width="465" alt="Screenshot 2024-08-19 at 11 27 18" src="https://github.com/user-attachments/assets/f6801143-696e-40c8-8168-e43c342a4ad9">

Thank you for your feedback,
Budiz

### Full version info

```shell
I cant enter Freecad
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@JohnOCFII any idea how to solve this ?

**Comment by JohnOCFII** [comment]
> @JohnOCFII any idea how to solve this ?

Sadly, no.  I just downloaded the latest build (38467) for macOS arm -- and it comes across as damaged.  I know that there were efforts over the weekend to change the way the bundle was created.  Wonder if it is related to that change?  See https://github.com/FreeCAD/FreeCAD-Bundle/issues/290
<img width="372" alt="Screenshot 2024-08-19 at 8 52 00 AM" src="https://github.com/user-attachments/assets/f3f0a6a1-6048-41bf-8bb2-6a07ac22b8e7">

**Comment by oursland** [comment]
@JohnOCFII Yes, this is likely due to #290.

**Comment by Syres916** [comment]
@oursland I think you meant to link https://github.com/FreeCAD/FreeCAD-Bundle/pull/290 rather than a FreeCAD PR from September 2016!!

**Comment by oursland** [comment]
@adrianinsaval When I try to debug the application it dies due to signal 9 (SIGKILL), which indicates an external process is terminating the execution.

<img width="608" alt="image" src="https://github.com/user-attachments/assets/e1a920f8-ef14-4697-a901-9c98fc5d36e0">

Is this perhaps a signing issue?

**Comment by oursland** [comment]
@Syres916 Yes, GitHub automatically linked my text and did so incorrectly.

**Comment by oursland** [comment]
I do believe it has to do with code signing or is somehow related.  I cannot manually launch the `freecad` binary.  When I move the file outside the `FreeCAD.app` directory, they begin to execute normally.

I did test this change extensively on my laptop for both https://github.com/FreeCAD/FreeCAD-Bundle/pull/290 and https://github.com/FreeCAD/FreeCAD-Bundle/pull/289, but I will admit I am less knowledgeable about macOS's requirements for running code outside the developer's machine and the related code-signing issues.
